File: class04/java/AP0401-OperadoresLogicos.java
import java.util.Scanner;
public class Voto {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Idade: "); int idade = sc.nextInt();
        System.out.print("Tem titulo? (s/n): "); boolean titulo = sc.next().equalsIgnoreCase("s");
        if (idade >= 16 && titulo) System.out.println("Pode votar");
        else System.out.println("Nao pode votar");
    }
}

File: class04/cpp/AP0401-OperadoresLogicos.cpp
#include <iostream>
int main() {
    int idade; char titulo;
    std::cout << "Idade: "; std::cin >> idade;
    std::cout << "Tem titulo? (s/n): "; std::cin >> titulo;
    if (idade >= 16 && (titulo == 's' || titulo == 'S'))
        std::cout << "Pode votar" << std::endl;
    else std::cout << "Nao pode votar" << std::endl;
    return 0;
}
